sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -98,6 +100,133 @@ public EditContactSteps(WebDriverHelpers webDriverHelpers) {
+              .assertThat(aContact.getRelationshipWithCase())
+              .isEqualToIgnoringCase(editedContact.getRelationshipWithCase());
+          softly
+              .assertThat(aContact.getDescriptionOfHowContactTookPlace())
+              .isEqualToIgnoringCase(editedContact.getDescriptionOfHowContactTookPlace());
+          softly
+              .assertThat(aContact.getProhibitionToWork())
+              .isEqualToIgnoringCase(editedContact.getProhibitionToWork());
+          softly
+              .assertThat(aContact.getHomeBasedQuarantinePossible())
+              .isEqualToIgnoringCase(editedContact.getHomeBasedQuarantinePossible());
+          softly
+              .assertThat(aContact.getQuarantine())
+              .isEqualToIgnoringCase(editedContact.getQuarantine());
+          softly.assertThat(aContact.getHighPriority()).isEqualToIgnoringCase("on");
+          softly
+              .assertThat(aContact.getDiabetes())
+              .isEqualToIgnoringCase(editedContact.getDiabetes());
+          softly
+              .assertThat(aContact.getImmunodeficiencyIncludingHiv())
+              .isEqualToIgnoringCase(editedContact.getImmunodeficiencyIncludingHiv());
+          softly
+              .assertThat(aContact.getLiverDisease())
+              .isEqualToIgnoringCase(editedContact.getLiverDisease());
+          softly
+              .assertThat(aContact.getMalignancy())
+              .isEqualToIgnoringCase(editedContact.getMalignancy());
+          softly
+              .assertThat(aContact.getChronicPulmonaryDisease())
+              .isEqualToIgnoringCase(editedContact.getChronicPulmonaryDisease());
+          softly
+              .assertThat(aContact.getRenalDisease())
+              .isEqualToIgnoringCase(editedContact.getRenalDisease());
+          softly
+              .assertThat(aContact.getChronicNeurologicalNeuromuscularDisease())
+              .isEqualToIgnoringCase(editedContact.getChronicNeurologicalNeuromuscularDisease());
+          softly
+              .assertThat(aContact.getCardiovascularDiseaseIncludingHypertension())
+              .isEqualToIgnoringCase(editedContact.getCardiovascularDiseaseIncludingHypertension());
+          softly
+              .assertThat(aContact.getAdditionalRelevantPreexistingConditions())
+              .isEqualToIgnoringCase(editedContact.getAdditionalRelevantPreexistingConditions());
+          softly
+              .assertThat(aContact.getVaccinationStatusForThisDisease())
+              .isEqualToIgnoringCase(editedContact.getVaccinationStatusForThisDisease());
+          softly
+              .assertThat(aContact.getImmunosuppressiveTherapy())
+              .isEqualToIgnoringCase(editedContact.getImmunosuppressiveTherapy());
+          softly
+              .assertThat(aContact.getActiveInCare())
+              .isEqualToIgnoringCase(editedContact.getActiveInCare());
+          softly.assertThat(aContact.getOverwriteFollowUp()).isEqualToIgnoringCase("on");
```

Comment:
Please change `overwriteFollowUp` into a boolean type because it is a checked or not checked option, and compare only objects not object vs value.

Reply:
reworked. added check as text for checbox's label for making sure we check the right one.